kazoo/tests/test_watchers.py

```diff
@@ -516,9 +517,10 @@ def changed(children):
             if counter > 0:
                 raise Exception("oops")
 
-        raises(Exception)(changed)
         counter += 1
-        self.client.create(self.path + '/' + 'smith')
+        with pytest.raises(Exception):
```

Comment:
Can this be changed to a more specific exception?

Reply:
Done

Reply:
Good catch, it caught some more errors. `handler.sleep()` -> `handler.sleep_func()`